adafruit_dymoscale.py

```diff
@@ -21,6 +21,7 @@
 """
 
 import time
+import microcontroller
```

Comment:
The CircuitPython interpreter doesn't actually store type annotations like other interpreters do, in an effort to save RAM on the microcontroller.  Therefore, it's also best if we can save RAM by not importing libraries used purely for type annotations.  Because it's okay to have things used only type annotations be undefined at runtime, we can use a `try`/`except` block guarded by something that's only available on a computer, like the `typing` library.  This way, the microcontroller won't end up importing anything in that block, whereas the computer will be able to successfully import everything!  If the `typing` library isn't used either we can disable a pylint error that would occur for adding it. 
 In this case, all the imports should look like this:

```python
import time
from pulseio import PulseIn
from micropython import const

try:
    import typing  # pylint: disable=unused-import
    from digitalio import DigitalInOut
    import microcontroller
except ImportError:
    pass
```

Let me know if you have any questions!